src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -139,6 +139,9 @@
     public static final String STRAND_BIAS_BY_SAMPLE_KEY =          "SB";
     public final static String TRANSMISSION_PROBABILITY_KEY =       "TP"; //PhaseByTransmission
     public static final String ZERO_COVERAGE_LOCI =                 "ZL"; //DiagnoseTargets
+    public static final String UNCONVERTED_BASE_COVERAGE_KEY =       "UNCONVERTED_BASE_COV";
```

Comment:
These keys are currently grouped with the other `FORMAT` keys, but they're used as `INFO` fields. They should be listed under the `INFO` keys group, but grouped together with a comment noting that they're methylation-related.